lib/csrf-hapi17.js

```diff
@@ -53,10 +52,7 @@ function csrfPlugin(server, options) {
           request.plugins[pkg.name] = { createToken };
         },
         verify: () => csrf.verify(request.headers[csrf.headerName], request.state[csrf.cookieName]),
-        continue: () => h.continue,
-        error: verify => {
-          throw new Boom(verify.error.message);
-        }
+        continue: () => h.continue
```

Comment:
If you revert this file so it still throws the boom error here and remove the `catch` in `csrf.js` we should be good to go